src/docs/development/ui/assets-and-images.md

```diff
@@ -31,20 +31,23 @@ specify the directory name with the `/` character at the end:
 ```yaml
 flutter:
   assets:
-    - assets/
+    - directory/
+    - directory/subdirectory/
 ```
 
-Note that only files located directly in the directory are
-included. To add files located in subdirectories,
-create an entry per directory.
+{{site.alert.note}}
+  That only files located directly in the directory are
```

Comment:
Delete "That", so:  "Only files..."

Reply:
Thank you sfshaza2 for always helping me correct my English.
Where are you from? It's 1am in my country, but I have a little insomnia and learn flutter for a while.

Reply:
Yeah, I often suffer from insomnia, too.  Though I seem to be sleeping better with the social isolation!